src/selectors/data/__specs__/bookingsSelectors.spec.js

```diff
@@ -971,3 +972,96 @@ describe('selectBookingByRouterMatch', () => {
     expect(booking).toStrictEqual({ id: 'AE', stockId: 'CG' })
   })
 })
+
+describe('selectPassedBookingsByOfferId', () => {
+  it('should return empty when no booking', () => {
+    // given
+    const state = {
+      data: {
+        offer: [{ id: 'A1' }],
+        stocks: [{ id: 'B1', offerId: 'A1' }],
+        bookings: [],
+      },
+    }
+
+    // when
+    const passedBookings = selectPassedBookingsByOfferId(state, 'A1')
+
+    // then
+    expect(passedBookings).toStrictEqual([])
+  })
+
+  it('should return empty when no bookings matching offer', () => {
+    // given
+    const state = {
+      data: {
+        offer: [{ id: 'A1' }],
+        stocks: [{ id: 'B1', offerId: 'A1' }, { id: 'B2', offerId: 'A1' }],
+        bookings: [{ id: 'C1', stockId: 'E1' }],
+      },
+    }
+
+    // when
+    const passedBookings = selectPassedBookingsByOfferId(state, 'A1')
+
+    // then
+    expect(passedBookings).toStrictEqual([])
+  })
+
+  it('should return not cancelled bookings in the passed', () => {
```

Comment:
```suggestion
  it('should return past bookings not cancelled', () => {
```

Vous m'avez fait mal avec "in the passed"

Reply:
La fatigue du vendredi peut être terrible parfois